### ISSUE TITLE
ProgressBar component color issues

### ISSUE THREAD

**Issue by AnkitaBagale** [razorpay/blade#2178]
Please check below issues in ProgressBar component:

1. Percentage text has black color. expected: gray.normal in dark mode.
2. Even though isIndeterminate is false, progress bar color is blinking.

code : 
```
<ProgressBar
  showPercentage
  isIndeterminate={false}
  size="large"
  variant="circular"
  value={(budget.settled_amount / budget.amount) * 100}
  color="positive"
/>
```

<img width="386" alt="image" src="https://github.com/razorpay/blade/assets/69760792/d6f1fb0f-7707-4f5b-9519-cee57a0c8e0e">

Design link - https://www.figma.com/design/uLiXKA43fEOUwU7Fw2eodL/Budgets?node-id=345-172373&m=dev

**Comment by snitin315** [comment]
@AnkitaBagale Thanks for the report, we will fix it on priority.

**Comment by snitin315** [comment]
> Percentage text has black color. expected: gray.normal in dark mode.

This will be fixed via #2185 

> Even though isIndeterminate is false, progress bar color is blinking.

For this, you need to set `isMeter={true}`, the `isIndeterminate` prop has no effect in case of circular progress.

**Comment by AnkitaBagale** [comment]
> > Percentage text has black color. expected: gray.normal in dark mode.
> 
> This will be fixed via #2185
> 
> > Even though isIndeterminate is false, progress bar color is blinking.
> 
> For this, you need to set `isMeter={true}`, the `isIndeterminate` prop has no effect in case of circular progress.

Thanks Nitin, will check this.
